js/src/toast.js

```diff
@@ -183,6 +183,7 @@ class Toast {
   // Private
 
   _positionToast() {
+    this._clearPositioning()
```

Comment:
Doesn't this change affect the current behavior though? I mean, we didn't clear their position previously? If it's only for our docs, maybe this shouldn't be here?

Reply:
It doesn't change the current behavior it improve it, because before that we cleared the positioning on hide, but with that change we can change position without hiding the toast